src/main/java/race/Utils/RandomGenerator.java

```diff
@@ -0,0 +1,24 @@
+package race.Utils;
+
+import java.util.Random;
+
+public class RandomGenerator implements RandomUtil {
+    Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
로 수정하면 더 안정적일 것 같아요 😄